client/app/bundles/course/survey/containers/SurveyFormDialogue/SurveyForm.jsx

```diff
@@ -68,158 +72,238 @@ const surveyFormTranslations = defineMessages({
   },
 });
 
-const validate = (values) => {
-  const errors = {};
+const validationSchema = yup.object({
+  title: yup.string().required(formTranslations.required),
+  description: yup.string(),
+  start_at: yup.date().nullable().required(formTranslations.required),
```

Comment:
Ditto - how does `nullable` and `required` work together?

Reply:
Explained above